.github/workflows/propagate_to_osbuild.yml

```diff
@@ -0,0 +1,63 @@
+name: Propagate update to OSBuild
+
+on: workflow_dispatch
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout manifest-db
+        uses: actions/checkout@v3
+        with:
+          path: manifest-db
+          fetch-depth: 0
+
+      - name: Update OSBuild
+        run: |
+          GH_REF="${{github.ref}}"
+          if [[ "$GH_REF" == *main* ]] ; then
+            # Get
+            MANIFEST_DB_HEAD="${{ github.sha }}"
+            echo "SHA: $MANIFEST_DB_HEAD"
+
+            # login as schutzbot
+            echo "${SCHUTZBOT_GH}" | gh auth login --with-token
+
+            # Clone osbuild
+            git clone https://github.com/osbuild/osbuild.git
+
+            cd osbuild
+            git config --local user.name "SchutzBot"
+            git config --local user.email "imagebuilder-bots+schutzbot@redhat.com"
+
+            git remote add upstream https://schutzbot:"$SCHUTZBOT_GH"@github.com/schutzbot/osbuild.git
+
+            now=$(date '+%Y-%m-%d-%H%M%S')
+            BRANCH_NAME="manifest-db-update-$now"
+            git checkout -b $BRANCH_NAME
+
+            # get the old ref from the OSBuild's Schutzfile
+            OLD_REF=$(jq -r '.global.dependencies."manifest-db".commit' Schutzfile)
+
+            # update only if the DB was updated since the version OSBuild is
+            # using
+            cd ../manifest-db
+            if [[ "$(git diff --name-only $GH_REF $OLD_REF manifest-db)" == *manifest-db* ]]; then
```

Comment:
I love your creative approach! No irony! We can do this in a simpler way though:

```
if ! git diff --quiet $GH_REF $OLD_REF manifest-db; then
```

See `--quiet` and `--exit-code` in [git-diff(1)](https://git-scm.com/docs/git-diff)

Reply:
Btw, can we use `MANIFEST_DB_HEAD` instead of `GH_REF` here? I think it might be more readable.

Reply:
Sure! Thanks for appreciating the creativity :smile:

Reply:
Yeah, using `GH_REF` was a working mistake.